egs/zeroth_korean/s5/local/online/run_nnet2_ms.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash
+
+# This is the "multi-splice" version of the online-nnet2 training script.
+# It's currently the best recipe.
+# You'll notice that we splice over successively larger windows as we go deeper
+# into the network.
+
+# Modified by Lucas Jo 2017 (Altas Guide)
+
+. cmd.sh
+
+
+stage=0
+train_stage=-10
+use_gpu=true
+dir=exp/nnet2_online/nnet_ms_a
+exit_train_stage=-100
+
+set -e
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if $use_gpu; then
+  if ! cuda-compiled; then
+    cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.  Otherwise, call this script with --use-gpu false
+EOF
+  fi
+  parallel_opts="--gpu 1"
+  num_threads=1
+  minibatch_size=512
+  # the _a is in case I want to change the parameters.
+else
+  # Use 4 nnet jobs just like run_4d_gpu.sh so the results should be
+  # almost the same, but this may be a little bit slow.
+  num_threads=16
+  minibatch_size=128
+  parallel_opts="--num-threads $num_threads"
+fi
+
+# do the common parts of the script.
+local/online/run_nnet2_common.sh --stage $stage
+
+echo "#### train_multiaplice_accel2.sh #####"
+if [ $stage -le 7 ]; then
+	#if [[ $(hostname -f) == *.clsp.jhu.edu ]] && [ ! -d $dir/egs/storage ]; then
+	#  utils/create_split_dir.pl \
+	#   /export/b0{3,4,5,6}/$USER/kaldi-data/egs/librispeech-$(date +'%m_%d_%H_%M')/s5/$dir/egs/storage $dir/egs/storage
+	#fi
+
+	# The size of the system is kept rather smaller than the run_7a_960.sh system:
+	# this is because we want it to be small enough that we could plausibly run it
+	# in real-time.
+	#--num-epochs 8 --num-jobs-initial 3 --num-jobs-final 18 \
+	#--initial-effective-lrate 0.0015 --final-effective-lrate 0.00015 \
+	#	--num-hidden-layers 6 \
+	#	--splice-indexes "layer0/-2:-1:0:1:2 layer1/-1:2 layer3/-3:3 layer4/-7:2" \
+	#	--num-hidden-layers 4 \
+	#	--splice-indexes "layer0/-1:0:1 layer1/-2:1 layer2/-4:2" \
+	#	--num-hidden-layers 3 \
+	#	--splice-indexes "layer0/-4:-3:-2:-1:0:1:2:3:4 layer2/-5:-1:3" \
+	steps/nnet2/train_multisplice_accel2.sh --stage $train_stage \
```

Comment:
I don't know why you'd include old nnet2 recipes in this PR.  The nnet3 recipes must be substantially better by now.  In any case you should be including results.